corehq/apps/users/tests/test_email_events.py

```diff
@@ -36,3 +40,72 @@ def test_bounced_event(self):
         invite = Invitation.objects.get(uuid=self.invite.uuid)
 
         self.assertEqual(invite.email_status, InvitationStatus.BOUNCED)
+
+
+class TestUserLimitWarnings(TestSnsEmailBase):
+    def setUp(self):
+        self.domain = 'test_domain'
+        self.recipients = ['orange@gmail.com', 'purple@gmail.com']
+        self.plan_limit = 100
+
+        get_admins_patcher = patch.object(WebUser, 'get_admins_by_domain')
+        self.mock_get_admins = get_admins_patcher.start()
+        self.mock_get_admins.return_value = [WebUser(username=self.recipients[0])]
+        self.addCleanup(get_admins_patcher.stop)
+
+        billing_admin_patcher = patch.object(WebUser, 'get_billing_admins_by_domain')
+        self.mock_get_billing_admins = billing_admin_patcher.start()
+        self.mock_get_billing_admins.return_value = [WebUser(username=self.recipients[1])]
+        self.addCleanup(billing_admin_patcher.stop)
+
+    def tearDown(self):
+        super().tearDown()
```

Comment:
If this method isn't doing anything but calling the parent, it should be removed. The testing framework will handle calling the `tearDown` method on its own.

Reply:
ah, I didn't know that tearDown is called automatically. Noted!